src/components/TextPage.jsx

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
 import BaseComponent from 'lib/BaseComponent';
+import TextPageLoad from 'transitions/TextPageLoad';
 
 export default class TextPage extends BaseComponent {
+  constructor (){
+    super();
+    this.state = {
+      loading: false,
+    };
+  }
+
+  componentDidMount () {
+    setTimeout(() => this.setState({ loading: true }), 300); 
+  }
+
   render () {
-    return (
-      <div className="text-page">
-        <h2 className="text-page__title">{this.props.title}</h2>
-        <div className="text-page__html" dangerouslySetInnerHTML={{__html: this.props.html}} />
-      </div>
-    );
+    const { loading } = this.state;
+    if (loading) {
+      return (
+        <div className="text-page">
+          <h2 className="text-page__title">{this.props.title}</h2>
+          <div className="text-page__html" dangerouslySetInnerHTML={{__html: this.props.html}} />
+        </div>
+      );
+    } else {
+      return (
+        <TextPageLoad />
```

Comment:
This should be in `TextPageController` on line 63 where it currently says `Loading`.

There should actually be no changes to this file since we aren't changing how the text page looks, just what renders while it's being loaded.